### PR TITLE
Disable travis tests for the client sub-project

### PR DESCRIPTION
**Pull request by urshunkeler**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,5 @@ matrix:
 script:
   - mvn -q -B -f jms-light/pom.xml clean verify
   - mvn -q -B -f kafka-connector/pom.xml clean verify
-  - mvn -q -B -f client/pom.xml clean verify -Dmaven.javadoc.skip=true -Dgpg.skip=true -Djava.util.logging.config.file=client/src/test/resources/logging.properties
```
Comment (denyska) [inline .travis.yml]: should we just keep ` mvn -q -B -f client/pom.xml clean verify -Dmaven.javadoc.skip=true -Dgpg.skip=true -DskipTests=true` ?

Comment (mprokhorenko) [inline .travis.yml]: +1 for -DskipTests=true

Comment (urshunkeler) [inline .travis.yml]: Changed.

**Comment by urshunkeler** [comment]
Somebody with write access will have to merge the pull request.

**Pull request merged**